### ISSUE TITLE
Adding extra slashes after domain name in url allows you to bypass JWT Authentication

### ISSUE THREAD

**Issue by gilz688** [tuupola/slim-jwt-auth#50]
The following array was passed to **JwtAuthentication** class constructor:
````
$options = [
    "secret" => $_ENV["JWT_SECRET"],
    "path" => ["/api/v1"],
    "passthrough" => ["/api/v1/login"]
]
````
Sending an HTTP request to "http://localhost/api/v1/restricted" returns HTTP 401 Unauthorized but for some reason I am able to bypass JWT authentication by adding one or more extra slashes after the domain name. e.g. "http://localhost//api/v1/restricted"

**Comment by tuupola** [comment]
Add a logger or error handler to debug why the the authentication fails.

https://github.com/tuupola/slim-jwt-auth#logger
https://github.com/tuupola/slim-jwt-auth#error

**Comment by gilz688** [comment]
HTTP 401 Unauthorized is expected because I made the request without the token in the Authorization header. (I forgot to mention it in the first comment.)

**Comment by tuupola** [comment]
Ok I misunderstood. It seems that even though `/foo` and `///foo` are different URL's for browser, on server side it seems to map to same url. A bit surprising behaviour but quickly checking  RFC 3986  this seems to be correct behaviour.

Will commit a fix soon.

**Comment by gilz688** [comment]
While waiting for your commit, I was able to temporarily fix the issue by replacing:
````$uri = str_replace("//", "/", $uri);```` with  ```$uri = preg_replace("/(\/)+/", "/", $uri);``` in **RequestPathRule.php** line 52.

**Comment by tuupola** [comment]
Yep, this is what I did: `$uri = preg_replace("#/+#", "/", $uri);`.

**Comment by tuupola** [comment]
Fixed in `2.3.2` and `1.0.4`.
